docs/Documentation/Systems/Swift/running.md

```diff
@@ -283,7 +287,7 @@ nm fhostone | grep intel | wc
 nm phostone | grep intel | wc
 ```
 
-on the two versions of the program. It will show how many calls to Intel routines are in each, 51 and 36 compared to 0 for the gnu versions.
+on the two versions of the program. It will show how many calls to Intel routines are in each, 51 and 36 compared to 0 for the gnu versions. (it gives all zeroes)
```

Comment:
You mention this has been fixed in your PR comments, should "(it gives all zeroes)" be removed?

Reply:
It is not fixed. I am just pointing out this issue.